com.unity.netcode.gameobjects/Tests/Runtime/Metrics/ServerLogsMetricTests.cs

```diff
@@ -11,17 +12,28 @@ namespace Unity.Netcode.RuntimeTests.Metrics
 {
     internal class ServerLogsMetricTests : SingleClientMetricTestBase
     {
-        // Header is dynamically sized due to packing, will be 2 bytes for all test messages.
-        private const int k_MessageHeaderSize = 2;
-        private static readonly int k_ServerLogSentMessageOverhead = 2 + k_MessageHeaderSize;
-        private static readonly int k_ServerLogReceivedMessageOverhead = 2;
+        // Header is dynamically sized due to packing, will be 3 bytes for all test messages.
```

Comment:
Note: This increased from 2 bytes to 3 bytes here because BytePacker.WriteValueBitPacked was changed to reserve 3 bits instead of 2 for a uint. It used to be that message length <= 63 would take 1 byte, now message length <= 31 does. The message length for a guid is consistently in between those sizes, so the message header for this test is now 3 bytes.